app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -8,6 +8,7 @@
     <li><a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md">follow component conventions</a></li>
   </ul>
   <p>See the <a href="https://github.com/alphagov/govuk_publishing_components">govuk_publishing_components gem</a> for further details, or <a href="https://docs.publishing.service.gov.uk/manual/components.html#component-guides">a list of all component guides</a>.</p>
+  <p>If you cannot find a component to fit your need, ask in #govuk-frontenders. If there is no suitable component, please use the <a href="https://design-system.service.gov.uk/styles/">Design System styles</a>.</p>
```

Comment:
I like the link to the design system docs, that's really good 👍 

While I think pointing people to `govuk-frontenders` is good, I'm wondering if it puts the onus on frontend devs to help instead of encouraging backend devs to explore the code and try and contribute? 
I think at some point a frontend dev may need to get involved, but perhaps we should try and guide people to explore other options first, e.g: trying to identify a component they could expand; raising a PR for review?